### ISSUE TITLE
tool bar hiding for presentation?

### ISSUE THREAD

**Issue by bernhardreiter** [spacedeck/spacedeck-open#67]
Tried to use the presentation mode. Probably I'm missing something, so this maybe a question or a feature request.

For presentation the presenter wants to hide the toolbar for the people watching.
How can this be archivied? Make the audience read-only participants? If so, how?

**Comment by mntmn** [comment]
Correct, there is currently no way to hide the toolbar, so that it is still possible to draw while presenting. We should introduce a toolbar hiding mode, but I don't know yet how to integrate it in the UI.

**Comment by mntmn** [comment]
See #104
